docs/Integrate GitOps with Secrets Management.md

```diff
@@ -301,84 +266,101 @@ oc adm policy add-scc-to-user privileged -z csi-secrets-store-provider-aws -n op
     2023/05/15 18:10:35 Updated Role policy for Role gitops-role-dev-aws-creds
     ```
 
-    e. Bind the service account with the role ARN by running the following command:
+    e. Check the role policy on AWS to confirm the <aws_region> of **Resource** in role policy matches the cluster region.
+    ```
+    {
+	"Version": "2012-10-17",
+	"Statement": [
+		{
+			"Effect": "Allow",
+			"Action": [
+				"secretsmanager:GetSecretValue",
+				"secretsmanager:DescribeSecret"
+			],
+			"Resource": "arn:aws:secretsmanager:<aws_region>:<aws_account_id>:secret:your-secret-xxxxxx"
+		}
+	]
+}
+    ```
+
+    f. Bind the service account with the role ARN by running the following command:
```

Comment:
@ciiay some formatting issue. Suggest to please check and rectify.

Reply:
@Srivaralakshmi Can you elaborate what exact formatting issue?